staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go

```diff
@@ -79,7 +81,15 @@ func getCallers(r interface{}) string {
 
 // ErrorHandlers is a list of functions which will be invoked when an unreturnable
 // error occurs.
-var ErrorHandlers = []func(error){logError}
+// TODO(lavalamp): for testability, this and the below HandleError function
+// should be packaged up into a testable and reusable object.
+var ErrorHandlers = []func(error){
+	logError,
+	(&rudimentaryErrorBackoff{
```

Comment:
I'm not a fan of this solution.  Other controllers use a backoff requeue mechanism in the work queue instead.  A global backoff handler for the entire process doesn't seem like a reasonable solution.